docs/source/user-guide/getting-started.rst

```diff
@@ -0,0 +1,28 @@
+***************
+Getting started
+***************
+
+Before starting the tutorials, consider reviewing
+:doc:`how to install and update conda build <../../install-conda-build>`
+and :doc:`conda build concepts <../../concepts/index>`.
+
+As you explore, you may find our :doc:`resources <../../resources/index>`
+collection helpful.
+
+.. _prereqs:
+
+Prerequisites
+=============
+
+Before starting the tutorials, we recommend that
```

Comment:
Since it's a Prereq, I'd remove "we recommend" and replace it with "you'll need to install..."